definitions/npm/bluebird_v3.x.x/flow_v0.32.x-/bluebird_v3.x.x.js

```diff
@@ -142,6 +142,7 @@ declare class Bluebird$Promise<R> {
   each<T, U>(iterator: (item: T, index: number, arrayLength: number) => Bluebird$Promisable<U>): Bluebird$Promise<Array<T>>;
   asCallback<T>(callback: (error: ?any, value?: T) => any, options?: Bluebird$SpreadOption): void;
   return<T>(value: T): Bluebird$Promise<T>;
+  spread<T>(...args: Array<T>): Bluebird$Promise<T>;
```

Comment:
What's the difference between `any` and `*`?

Pushed the change though 😄  And expanded the test so it actually failed like you thought it would before using `*`